robot_introspection:
  ros__parameters:
    robot_id: 2 
    robot_name: "Minibot 2"
    battery_capacity: 50.0    #mah
    battery_cells_number: 2
    battery_charge_percentage: 80 #%
    battery_supply_status: "NOT CHARGING"